multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -62,13 +62,13 @@ type (
 	// ServiceExportReconciler reconciles a ServiceExport object in the member cluster.
 	ServiceExportReconciler struct {
 		client.Client
-		Scheme                  *runtime.Scheme
-		remoteCommonAreaManager *commonarea.RemoteCommonAreaManager
-		installedSvcs           cache.Indexer
-		installedEps            cache.Indexer
-		leaderNamespace         string
-		leaderClusterID         string
-		localClusterID          string
+		Scheme          *runtime.Scheme
+		mcReconciler    *MemberClusterSetReconciler
```

Comment:
The point of having `RemoteCommonAntreaManagerGetter` is to expose only necessary methods instead of all methods to `ServiceExportReconciler`, using concrete type here makes the interface pointless.

Reply:
Fix with passing `RemoteCommonAreaGetter`.